### DISCUSSION TITLE
Welcome to spring-petclinic-microservices Discussions!

### DISCUSSION THREAD

**Discussion by arey** [spring-petclinic/spring-petclinic-microservices#194] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by feli0821** [comment]
Hello, I am trying to spin up a docker.  I am wondering how to install this in my windows docker-compose?
Appreciate simple steps as I am very new to dockers.  Thank you.

**Reply by feli0821** [reply]
Btw, this is a sample error from one customer-service docker:
2022/01/12 09:34:42 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:48 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:49 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:50 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:51 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:52 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:53 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:54 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:55 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:56 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:57 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:58 Problem with dial: dial tcp 172.22.0.6:8761: connect: connection refused. Sleeping 1s
2022/01/12 09:34:58 Timeout after 1m0s waiting on dependencies to become available: [tcp://discovery-server:8761]

**Comment by arey** [comment]
Hi @feli0821 could you please open a new thread for this discussion?
